orc8r/cloud/go/services/certifier/obsidian/handlers/handlers.go

```diff
@@ -172,32 +177,34 @@ func loginHandler(c echo.Context) error {
 	if err != nil {
 		return obsidian.MakeHTTPError(err, http.StatusInternalServerError)
 	}
-	return c.JSON(http.StatusOK, res.Policies)
+	return c.JSON(http.StatusOK, res.PolicyLists)
 }
 
-func resourcesModelToProto(resources *models.Resources) *protos.ResourceList {
-	resourceList := make([]*protos.Resource, len(*resources))
-	for i, resource := range *resources {
-		resourceProto := &protos.Resource{
-			Effect:       matchEffect(resource.Effect),
-			Action:       matchAction(resource.Action),
-			ResourceType: matchResourceType(resource.ResourceType),
-			Resource:     resource.Resource,
+func policiesModelToProto(policies *models.Policies) ([]*protos.Policy, error) {
+	resourceList := make([]*protos.Policy, len(*policies))
+	for i, resource := range *policies {
+		resourceProto := &protos.Policy{
+			Effect: matchEffect(resource.Effect),
+			Action: matchAction(resource.Action),
+		}
+		if err := setResource(resource, resourceProto); err != nil {
+			return nil, err
 		}
 		resourceList[i] = resourceProto
 	}
-	return &protos.ResourceList{Resources: resourceList}
+	return resourceList, nil
```

Comment:
We're using "resource" and "resourceList" but returning "policy" and "policies". Think should all be the latter (?)